cran-comments.md

```diff
@@ -8,3 +8,6 @@
 
 ## R CMD check results
 0 errors | 0 warnings | 0 notes
+
+## Authors and copyright holders
```

Comment:
Add a “Resubmission” section at the top of cran-comments.md. This should clearly identify that the package is a resubmission, and list the changes that you made. Example:

## Resubmission
This is a resubmission. In this version I have:

* Converted the DESCRIPTION title to title case.

* More clearly identified the copyright holders in the DESCRIPTION
  and LICENSE files.

Reply:
Check out the "**On failure**" section
http://r-pkgs.had.co.nz/release.html

Reply:
I'll add the changes to cran-comments.md in a separate PR

Reply:
@heemanshusuri you can remove your changes to this file

Reply:
okay, sounds good.